cinder-{{cookiecutter.driver_name_lc}}/unit_tests/test_lib_charm_openstack_cinder_{{cookiecutter.driver_name_lc}}.py

```diff
@@ -12,38 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from __future__ import absolute_import
-from __future__ import print_function
+import unittest
+from src.charm import Cinder{{ cookiecutter.driver_name }}Charm
+from ops.model import ActiveStatus
+from ops.testing import Harness
 
-import charmhelpers
 
-import charm.openstack.cinder_{{ cookiecutter.driver_name_lc }} as cinder_{{ cookiecutter.driver_name_lc }}
```

Comment:
Also the name of this file `test_lib_charm_openstack_...` doesn't make sense anymore as it used to refer to `charm.openstack.` and should thus be renamed

Reply:
Fixed, thanks